copyright/README.md

```diff
@@ -0,0 +1,30 @@
+## Mobify Copyright Headers
+
+This tool reads from the `headers/copyright.extension.txt` text files to select what header to insert, where `extension` is the target file extension
+
+## Quick Start
+
+- Clone this repo
```

Comment:
@jeremywiebe so ideally would we want something like 
```bash
npm install mobify-code-style
npm run lint:copyright
```

Or something along these lines?

I made the change in the README right now - so I should also add something like

```bash
"lint:copyright": "node_modules/mobify-code-style/copyright/copyright.js PATH --lint"
```

to your `package.json` - or is there an easier way than digging into the node modules in the npm script 😛 🤔